terms/syntheticfeatures_fcm.tex

```diff
@@ -0,0 +1,112 @@
+\chapter{Synthetic Features}
+\label{ch:synthetic features}\index{synthetic-features}
+
+\section{The problem}
+When a data scientist is looking at employing a machine learning technique for some problem at hand, it is important for the scientist to experiment, evaluate and test various algorithms and techniques so as to find the one that performs the best for that particular context.
+
+This comes with a number of challenges, however probably one of the main challenges is sourcing the dataset to be used as part of the evaluation process \citep{patki2016}. In situations where the scientist already has access to such data, this would generally not be an issue, however it often happens that the researcher would not have such access available. Moreover, even when data is available, it is not always clean enough to provide good training.
+
+To mitigate this problem, a possible solution is to generate and use synthetic features. This gives the researcher the freedom to generate data that fits the scope of the problem at hand, without access to the original data, whilst still allowing for proper evaluation of different techniques and algorithms \citep{patki2016}.
+
+Over and above to unavailability of data, such an approach can also be used to enhance and augment limited data that might be available, such as synthetic over-sampling in case of unbalanced datasets \citep{tallo2018}.
+
+Such an approach presents itself with a number of advantages for evaluation models and algorithms:
+
+\begin{itemize}
+    \item Evaluation of the model’s behaviour and performance under specific data conditions and complexities, and not just over the standard statistical distribution of a real dataset.
```

Comment:
Have you read this to check its fine?  Look at what is generated for `model’s`.    You'll see this is broken in the pdf.  Fix.